nautobot/dcim/views.py

```diff
@@ -240,6 +243,7 @@ class LocationTypeUIViewSet(NautobotUIViewSet):
     form_class = forms.LocationTypeForm
     bulk_update_form_class = forms.LocationTypeBulkEditForm
     serializer_class = serializers.LocationSerializer
+    breadcrumbs = LocationsBreadcrumbs()
```

Comment:
This "looks" wrong. Maybe rename `Location[s]Breadcrumbs` to something more generic like `AncestorBreadcrumbs`?